sdk/servicebus/service-bus/src/connectionContext.ts

```diff
@@ -354,47 +397,45 @@ export namespace ConnectionContext {
         await Promise.all(detachCalls);
       }
 
+      // Calling onDetached on batching receiver and session receiver for the same reasons as sender
+      if (!state.wasConnectionCloseCalled && state.numReceivers) {
+        logger.verbose(
+          `[${connectionContext.connection.id}] connection.close() was not called from the sdk and there were ${state.numReceivers} ` +
```

Comment:
Fix numbers accordingly or move this log to the callOnDetachedOnReceivers

Reply:
Is the inaccuracy because the numReceivers is both session and non-session receivers?

Reply:
Yes. (And batching and streaming.)